x-pack/plugins/lens/public/indexpattern_datasource/indexpattern.tsx

```diff
@@ -281,7 +281,6 @@ export function getIndexPatternDatasource({
     ) => {
       render(
         <LayerPanel
-          state={props.state}
```

Comment:
state is overriden by props https://github.com/elastic/kibana/pull/67666/files#diff-34ecffdffefb6fb82cf266415012e018R295